bootcamps/gamedevonline.html

```diff
@@ -143,8 +158,68 @@ <h3 class="uppercase color-primary mb40 mb-xs-24">Overview</h3>
 
 		        </div>
 
+            <section class="bg-secondary">
```

Comment:
There is white corners around photos. Make then transparent. 
<img width="312" alt="screen shot 2017-04-03 at 3 52 05 pm" src="https://cloud.githubusercontent.com/assets/1327050/24605207/8cad6958-1885-11e7-9af3-a37a8fae2b59.png">